src/main/java/org/springframework/cloud/deployer/spi/cloudfoundry/CloudFoundryAppDeployer.java

```diff
@@ -167,18 +165,18 @@ private AppStatus createAppStatus(ApplicationDetail applicationDetail, String de
 		return builder.build();
 	}
 
-	private AppStatus createEmptyAppStatus(String deploymentId) {
-		return AppStatus.of(deploymentId)
+	private AppStatus createEmptyAppStatus(String applicationName) {
```

Comment:
This really is a deploymentId (which happens to be an application name, but at several places, deploymentId is what we're talking about...)